add_new_to_compendia.snakefile

```diff
@@ -104,16 +108,68 @@ rule salmon:
         --minScoreFraction 0.65 --fldMean 51 --seqBias -l A    \
         -r {input.reads} -o {params.out_dir}
     '''
+
+rule interleave_reads:
+    """
+    Since paired-end reads need to be processed as single-end, interleave reads
+    using the same code as is in the download script for publicly available samples
+    No conda env is required for this rule since everything is installed in the
+    `sputum` env
+    Rule assumes that input files are already in place at inputs/raw/hogan/
+    """ 
+    output: "outputs/interleaved_spu/{spu}.fastq.gz"
+    params: indir = "inputs/raw/hogan/"
+    threads: 1
+    resources: mem_mb = 4000
+    run:
+        # use metadata to determine input file names, which have additional characters
+        # over what is in the spu wildcard
+        row = h.loc[h['sample'] == wildcards.spu]
+        r1 = row['r1'].values[0]
+        r2 = row['r2'].values[0]
+        shell("reformat.sh in1={params.indir}{r1} in2={params.indir}{r2} out={output}")
     
+rule salmon_spu:
+    """
+    This is the same rue as above, with the only difference being that this
```

Comment:
```suggestion
    This is the same rule as above, with the only difference being that this
```

Reply:
Also maybe you could refer to the specific rule here? i.e. "This is the same as the 'salmon' rule above, with the only difference..."

Reply:
```suggestion
    This is the same rule as above (rule `salmon`), with the only difference being that this
```